cli/medperf/entities/benchmark.py

```diff
@@ -175,27 +187,30 @@ def todict(self) -> dict:
         dict: Dictionary containing benchmark information
         """
         return {
-            "uid": self.uid,
+            "id": self.uid,
```

Comment:
I think one of the next important steps is to somehow share this dictionary description of each entity so that we only have to specify it once. Right now we have the same structure copied multiple times across entities and commands. This is very error-prone. This is not specific to this PR, but something I realize we should work on soon given how this dictionaries increased in size.

Reply:
Exactly.

I tried to avoid unintended future errors by designing the commands to directly call the entities whenever the command's `todict` is used (specifically inside `command.upload` and `command.write`). So currently entities constructors are making sure to have unified dictionaries between commands and entities. But yes, it's not the perfect way. I have added in this PR the expected dictionaries in `tests.mocks.requests`, maybe we can start from there...